fedmsg.d/sse.py

```diff
@@ -0,0 +1,12 @@
+config = {
+
+    # SSE
```

Comment:
yes this is more about rabbitmq but this is directly related to sse use of rabbitmq